rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h

```diff
@@ -32,6 +32,10 @@
 #include "rcl_yaml_param_parser/types.h"
 #include "rcl_yaml_param_parser/visibility_control.h"
 
+#ifdef _WIN32
+#define strcasecmp stricmp
+#endif
+
```

Comment:
I think that there is no standard case insensitive string compare in C. but I am negative on this, this will cause complication and dependency especially for maintenance. (at least i believe that this is not the right place.) Instead of this, we would create `rcutils_strcasecmp` based on `strcasecmp` and `stricmp`? Any thoughts?